ocppj/queue.go

```diff
@@ -145,18 +145,15 @@ func (f *FIFOQueueMap) Init() {
 }
 
 func (f *FIFOQueueMap) Get(clientID string) (RequestQueue, bool) {
-	f.mutex.Lock()
-	defer f.mutex.Unlock()
-	q, ok := f.data[clientID]
-	return q, ok
+	f.mutex.RLock()
+	defer f.mutex.RUnlock()
+	return f.data[clientID]
```

Comment:
This doesn't compile sadly. Is it even supported in any Go version?

Reply:
Sorry, did these changes online- fixed.